scripts/epa/airdata/criteria_gases/criteria_gases.py

```diff
@@ -0,0 +1,196 @@
+]
+
+# Template MCF for StatVarObservation
+TEMPLATE_MCF = '''
+Node: E:EPA_CriteriaGases->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: C:EPA_CriteriaGases->Mean_SV
+measurementMethod: C:EPA_CriteriaGases->Method
+observationDate: C:EPA_CriteriaGases->Date
+observationAbout: E:EPA_CriteriaGases->E0
+observationPeriod: dcs:"P1D"
+value: C:EPA_CriteriaGases->Mean
+unit: C:EPA_CriteriaGases->Units
+
+Node: E:EPA_CriteriaGases->E2
+typeOf: dcs:StatVarObservation
+variableMeasured: C:EPA_CriteriaGases->Max_SV
+measurementMethod: C:EPA_CriteriaGases->Method
+observationDate: C:EPA_CriteriaGases->Date
+observationAbout: E:EPA_CriteriaGases->E0
+observationPeriod: dcs:"P1D"
+value: C:EPA_CriteriaGases->Max
+unit: C:EPA_CriteriaGases->Units
+
+Node: E:EPA_CriteriaGases->E3
+typeOf: dcs:StatVarObservation
+variableMeasured: C:EPA_CriteriaGases->AQI_SV
+measurementMethod: C:EPA_CriteriaGases->Method
+observationDate: C:EPA_CriteriaGases->Date
+observationAbout: E:EPA_CriteriaGases->E0
+observationPeriod: dcs:"P1D"
+value: C:EPA_CriteriaGases->AQI
+'''
+
+# Template MCF for Air Quality Site
+TEMPLATE_MCF_AIR_QUALITY_SITE = '''
+Node: E:EPA_CriteriaGases->E0
+typeOf: dcs:AirQualitySite
+dcid: C:EPA_CriteriaGases->Site_Number
+name: C:EPA_CriteriaGases->Site_Name
+location: C:EPA_CriteriaGases->Site_Location
+containedInPlace: C:EPA_CriteriaGases->County
+airQualitySiteMonitor: C:EPA_CriteriaGases->POC
+'''
+
+
+# Convert to CamelCase (splitting on spaces)
+def get_camel_case(s):
+    if s == '' or s == ' - ':
+        return ''
+    parts = s.lower().split()
+    result = ''
+    for i in range(len(parts)):
+        result += parts[i][0].upper()
+        if len(parts[i]) > 0:
+            result += parts[i][1:]
+    return result
+
+
+def get_pollutant_standard(s):
+    return s.replace(' ', '_').replace('-', '')
+
+
+# Hardcoded for consistency in StatisticalVariable names
+def get_suffix(parameter):
+    if parameter == '44201':
+        return 'Ozone'
+    elif parameter == '42401':
+        return 'SO2'
+    elif parameter == '42101':
+        return 'CO'
+    elif parameter == '42602':
+        return 'NO2'
+    return ''
+
+
+def create_csv(csv_file_path):
+    with open(csv_file_path, 'w', newline='') as f_out:
+        writer = csv.DictWriter(f_out,
+                                fieldnames=CSV_COLUMNS,
+                                lineterminator='\n')
+        writer.writeheader()
+
+
+def write_csv(csv_file_path, reader):
+    with open(csv_file_path, 'a', newline='') as f_out:
+        writer = csv.DictWriter(f_out,
+                                fieldnames=CSV_COLUMNS,
+                                lineterminator='\n')
+        keys = set()
+        for observation in reader:
+            # Select only one observation for a given date, site, and pollutant standard
+            # POC will be stored as airQualitySiteMonitor
+            key = '{date}{state}{county}{site}{standard}'.format(
```

Comment:
just curious... do we need {state}{county} or is {site} unique?

Reply:
as I learned the hard way, unfortunately the site code is only within a county, so we need state and county to disambiguate